lectures/_toc.yml

```diff
@@ -12,22 +12,25 @@ parts:
   - file: business_cycle
   - file: inflation_history
   - file: inequality
-- caption: Essential Tools
+- caption: Foundations
   numbered: true
   chapters:
-  - file: complex_and_trig
-  - file: linear_equations
-  - file: eigen_I
   - file: intro_supply_demand
-- caption: Linear Dynamics
+  - file: linear_equations
+  - file: complex_and_trig
+- caption: "Linear Dynamics: Finite Horizons"
   numbered: true
   chapters:
   - file: pv
   - file: cons_smooth
   - file: equalizing_difference
   - file: cagan_ree
   - file: cagan_adaptive
+- caption: "Linear Dynamics: Infinite Horizons"
+  numbered: true
+  chapters:
   - file: geom_series
```

Comment:
- [ ] `geom_series` this should be moved above `Foundations` (cc @thomassargent30) after `complex_and_trig`